uni/UWaterloo.py

```diff
@@ -123,23 +122,24 @@ def scrapeCourseList(self, uw, term, subjectList):
         courseList = []
         # For each subject scrape courses
         for subject in subjectList:
-
             # Gets all courses based on term and subject
             for course in uw.term_subject_schedule(term, subject['subject']):
 
                 if course['catalog_number'] != prevClass or prevClass == '':
                     group = []
+                    # initialize course dictionary
+                courseDict = {'coursenum': course['catalog_number'], 'subject': subject['subject'],
+                              'term': term,
+                              'id': course['class_number'], 'type': course['section'][:3], 'group': [],
+                              'location': course['campus'], 'curEnroll': course['enrollment_capacity'],
+                              'rooms': [course['classes'][0]['location']['building'], course['classes'][0]['location']['room']],
```

Comment:
What if there are multiple rooms for the class?
Why not from a single `room` string from the building and room?